compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -28,6 +28,10 @@
 #include "mlir/IR/Matchers.h"
 #include "mlir/Transforms/GreedyPatternRewriteDriver.h"
 
+#define DEBUG_TYPE "kernel-dispatch"
+#define DBGS() (llvm::dbgs())
+#define KD_DBGS() (DBGS() << '[' << DEBUG_TYPE << "] ")
```

Comment:
The `DBGS` is not used by others. Maybe combine two macro and use `DBGS`?

```suggestion
#define DEBUG_TYPE "kernel-dispatch"
#define DBGS() (llvm::dbgs()<< '[' << DEBUG_TYPE << "] ")
```

Reply:
I can remove `DBGS` for now but it will be needed when we have to print a message with multiple lines without the pass banner. `KD_DBGS` name is to state that the message will have the `[kernel-dispatch]` banner.